test/e2e/e2e.go

```diff
@@ -0,0 +1,234 @@
+	channels := clients.Eventing.EventingV1alpha1().Channels(pkgTest.Flags.Namespace)
+	res, err := channels.Create(channel)
+	if err != nil {
+		return err
+	}
+	cleaner.Add(v1alpha1.SchemeGroupVersion.Group, v1alpha1.SchemeGroupVersion.Version, "channels", pkgTest.Flags.Namespace, res.ObjectMeta.Name)
+	return nil
+}
+
+// CreateSubscription will create a Subscription
+func CreateSubscription(clients *test.Clients, subs *v1alpha1.Subscription, logger *logging.BaseLogger, cleaner *test.Cleaner) error {
+	subscriptions := clients.Eventing.EventingV1alpha1().Subscriptions(pkgTest.Flags.Namespace)
+	res, err := subscriptions.Create(subs)
+	if err != nil {
+		return err
+	}
+	cleaner.Add(v1alpha1.SchemeGroupVersion.Group, v1alpha1.SchemeGroupVersion.Version, "subscriptions", pkgTest.Flags.Namespace, res.ObjectMeta.Name)
+	return nil
+}
+
+// CreateServiceAccount will create a service account
+func CreateServiceAccount(clients *test.Clients, sa *corev1.ServiceAccount, logger *logging.BaseLogger, cleaner *test.Cleaner) error {
+	sas := clients.Kube.Kube.CoreV1().ServiceAccounts(pkgTest.Flags.Namespace)
+	res, err := sas.Create(sa)
+	if err != nil {
+		return err
+	}
+	cleaner.Add(corev1.SchemeGroupVersion.Group, corev1.SchemeGroupVersion.Version, "serviceaccounts", pkgTest.Flags.Namespace, res.ObjectMeta.Name)
+	return nil
+}
+
+// CreateClusterRoleBinding will create a service account binding
+func CreateClusterRoleBinding(clients *test.Clients, crb *rbacV1beta1.ClusterRoleBinding, logger *logging.BaseLogger, cleaner *test.Cleaner) error {
+	clusterRoleBindings := clients.Kube.Kube.RbacV1beta1().ClusterRoleBindings()
+	res, err := clusterRoleBindings.Create(crb)
+	if err != nil {
+		return err
+	}
+	cleaner.Add(rbacV1beta1.SchemeGroupVersion.Group, rbacV1beta1.SchemeGroupVersion.Version, "clusterrolebindings", "", res.ObjectMeta.Name)
+	return nil
+}
+
+// CreateServiceAccountAndBinding creates both ServiceAccount and ClusterRoleBinding with default
+// cluster-admin role
+func CreateServiceAccountAndBinding(clients *test.Clients, name string, logger *logging.BaseLogger, cleaner *test.Cleaner) error {
+	sa := &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: pkgTest.Flags.Namespace,
+		},
+	}
+	err := CreateServiceAccount(clients, sa, logger, cleaner)
+	if err != nil {
+		return err
+	}
+	crb := &rbacV1beta1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "e2e-tests-admin",
+		},
+		Subjects: []rbacV1beta1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      name,
+				Namespace: pkgTest.Flags.Namespace,
+			},
+		},
+		RoleRef: rbacV1beta1.RoleRef{
+			Kind:     "ClusterRole",
+			Name:     "cluster-admin",
+			APIGroup: "rbac.authorization.k8s.io",
+		},
+	}
+	err = CreateClusterRoleBinding(clients, crb, logger, cleaner)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
Do we not want these cleaned up? As in, how come these are not added to the cleaner?

Reply:
They are already added to the cleaner from the `CreateServiceAccount` and `CreateClusterRoleBinding` methods called here.